pyeapi/utils.py

```diff
@@ -243,6 +243,24 @@ class CliVariants:
     or with 2 or more sequences of cli (or a mix of list and str types), e.g.:
         ``CliVariants( ['new cli1', 'new cli2'], 'alt cli3', 'legacy cli4' )``
     """
+    @staticmethod
+    def expand( cmds ):
+        """cnds is a list of str and CliVariants, this method returns a list
```

Comment:
typo here? should this be cmds instead of cnds?

Reply:
yes, sharp eye, will fix this typo.